app/scripts/components/common/nav-wrapper.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import T from 'prop-types';
+import styled from 'styled-components';
+import PageHeader from './page-header';
+
+import PageLocalNav from '$components/common/page-local-nav';
+import useScrollDirection from '$utils/use-scroll-direction';
+
+const NavWrapper = styled.div`
+  position: sticky;
+  top: 0;
+  z-index: 1000;
+`;
+
+const AnimatedPageHeader = styled(PageHeader)`
+  transition: margin-top 0.32s ease-out;
+  margin-top: 0;
+  &.up {
+    margin-top: 0;
+  }
+  &.down {
+    margin-top: calc(
+      -2.5rem - (1rem * var(--base-space-multiplier, 1) * 0.75 * 2)
```

Comment:
Can you add a little comment explaining the values, namely the `0.75 * 2`

Reply:
I covered here: https://github.com/NASA-IMPACT/delta-ui/pull/207/files#diff-53f55889f079d886355da3a9f5d993dd6052bafba682ac471d3fe410ec41743dR71 (Also I could use variableGlsp instead of raw value so it is a win ✌️ - couldn't get the nested calc before!)